app/components/edit-reminder.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions:  {
+    editReminder() {
+      console.log("Button does thing!");
```

Comment:
Looks all this file does is console log some stuff...which we all know doesn't get merged to master :)